core/plugins/org.polarsys.capella.core.sirius.analysis/src/org/polarsys/capella/core/sirius/analysis/preferences/TitleBlockPreferencesInitializer.java

```diff
@@ -14,73 +14,53 @@
 
 import org.eclipse.jface.preference.IPreferenceStore;
 import org.polarsys.capella.core.commands.preferences.service.AbstractPreferencesInitializer;
+import org.polarsys.capella.core.commands.preferences.service.ScopedCapellaPreferencesStore;
 import org.polarsys.capella.core.platform.sirius.ui.actions.CapellaActionsActivator;
-import org.polarsys.capella.core.preferences.Activator;
 
 public class TitleBlockPreferencesInitializer extends AbstractPreferencesInitializer {
 
-  public static final String TABEL_CONTENT_PREFERENCE_STORE = "tableTitleBlock";
-  public static final String COLUMNS_NUMBER_PREFERENCE_STORE = "columnsNumberTitleBlock";
-  public static final String LINES_NUMBER_PREFERENCE_STORE = "linesNumberTitleBlock";
-  public static final String DEFAULT_TITLEBLOCK_PREFERENCE_STORE = "defaultTitleBlock";
-  public static final String SEPARATOR = "SEPARATOR";
-  public static final String DEFAULT_TABLE = "Name" + SEPARATOR + "feature:name" + SEPARATOR + "Synchronized" 
-  + SEPARATOR + "aql:self.representation.synchronized" + SEPARATOR + "Summary" + SEPARATOR + "aql:self.target.summary" + SEPARATOR
-  + "Description" + SEPARATOR + "feature:documentation";
-
-  
   public TitleBlockPreferencesInitializer() {
     super(CapellaActionsActivator.PLUGIN_ID);
   }
 
   @Override
   public void initializeDefaultPreferences() {
-    IPreferenceStore preferenceStore = doGetPreferenceStore();
-    preferenceStore.setDefault(COLUMNS_NUMBER_PREFERENCE_STORE, 2);
-    preferenceStore.setDefault(LINES_NUMBER_PREFERENCE_STORE, 2);
-    preferenceStore.setDefault(TABEL_CONTENT_PREFERENCE_STORE, DEFAULT_TABLE);
-
-    preferenceStore.setDefault(DEFAULT_TITLEBLOCK_PREFERENCE_STORE, false);
-  }
-  
-  
-  /**
-   * @return IPreferenceStore the default preference store instance
-   */
-  public static IPreferenceStore doGetPreferenceStore() {
-    return Activator.getDefault().getPreferenceStore();
+    IPreferenceStore preferenceStore = ScopedCapellaPreferencesStore.getInstance(CapellaActionsActivator.PLUGIN_ID);
+    preferenceStore.setDefault(TitleBlockPreferencePage.COLUMNS_NUMBER_PREFERENCE_STORE, 2);

```

Comment:
Just for reminder:
org.polarsys.capella.core.commands.preferences.service.AbstractPreferencesInitializer are inconsistent and not working properly.

`putInt` If the InstanceScope contains a value, at restart the default value will contains the previous instance value, not the one given in parameter. It will never set only the default value. always in instance scope.

`putBoolean` and `putString` used in https://github.com/eclipse/capella/blob/6ef8e899f3503c655a1c528a2d33442dc722e90d/core/plugins/org.polarsys.capella.core.model.preferences/src/org/polarsys/capella/core/model/preferences/DeletePreferences.java#L35 do nothing if DefaultScope.

If ProjectScope, like used in initializer, as there is never a selected project at startup, its also weird.